### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Andrew-Shortridge** [cooperrc/computational-mechanics#15] [Q&A]
My name is Andrew Shortridge, and I am a senior mechanical engineering student. One interesting fact about me is that I am finishing up a professional hockey career right now and plan on going back to school full-time in order to finish my undergraduate degree and start my PhD in ME at UConn. I'm sure most of you know that this is a required class for ME, but I am extremely excited on being able to build skills with this class that I can use for my future research in the Spring of 2022. Overall, this will hopefully help me build a depth of computational knowledge that I will be able to use in my research and in my future career as well.

**Comment by tempcooper** [comment]
> professional hockey career right now and plan on going back to school full-time in order to finish my undergraduate degree and start my PhD in ME at UConn

:ice_hockey: Awesome! Thanks for joining another Cooper-course @Andrew-Shortridge ! You'll recognize the introductory Python content from ME5180, but review never hurts. 

Its great to hear you're coming back to UConn for PhD!! We are lucky to have you. 

> I am extremely excited on being able to build skills with this class

me too :+1:
